Appsilon/rhino
# Feat: Build js sass in ci

### Changes
Closes #92 

* Adds Build JavaScript and Build Sass to CI.
* Bumps dev version to 1.1.1.9004.
* Updates `NEWS` about the new build steps in the template workflow.

### How to test
* [ Example workflow](https://github.com/Appsilon/rhino-showcase/actions/runs/3278440602/jobs/5396828885) where `rhino::lint_sass()` passed but `rhino::build_sass()` failed
* Example workflow where `rhino::build_js()` failed. Note: I was not able to make a JS that will pass the lint but fail the build, so this is more proof of concept that `rhino::build_js()` will throw an error in CI if it fails.
* Later in the workflow, Cypress test fails. I do not know why, but this should not affect or negate the fails for building sass and js.
